site/docs/releases.md

```diff
@@ -71,6 +71,88 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
   ...
 </dependencies>
 ```
+
+### 1.6.0 release
+Apache Iceberg 1.6.0 was released on July 23, 2024.
+
+The 1.6.0 release contains fixes and dependency updates.
```

Comment:
Also some features, right? Like the new Kafka connect etc?

Reply:
That's right, let me rephrase.